src/Contract/Operation/Unfoldable.php

```diff
@@ -14,11 +14,12 @@ interface Unfoldable
      * @see https://loophp-collection.readthedocs.io/en/stable/pages/api.html#unfold
      *
      * @template T
+     * @template TKey
```

Comment:
`TKey` is unused here, did you mean to change the return type on line 22 to `Collection<TKey, iterable<array-key, T>>`?